examples/gno.land/r/moul/config/config_test.gno

```diff
@@ -0,0 +1 @@
+package config
```

Comment:
Leftover file?

Reply:
It's to ensure that `gno test` does not skip the folder. Although there are no unit tests, it still passes through the gnovm syntax checks and other validations.